pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "OpenFisca-France"
-version = "169.3.1"
+version = "169.4.1"
```

```suggestion
version = "169.4.0"
``` ou 169.3.2, j'ai l'impression qu'il n'y a pas encore de 169.4.0 si je ne m'abuse. Idem dans le Changelog.